pandas/io/formats/printing.py

```diff
@@ -283,9 +284,14 @@ def format_object_summary(obj, formatter, is_justify=True, name=None):
         string formatter for an element
     is_justify : boolean
         should justify the display
-    name : name, optiona
+    name : name, optional
         defaults to the class name of the obj
 
+        Pass ``False`` to indicate that subsequent lines should
```

Comment:
what calls this with False? IOW what has a name that u don’t want to print

Reply:
can this be another parameter then? it seems like it is used for 2 purposes

Reply:
Done. Calling it `indent_for_name`.